nautobot/core/api/fields.py

```diff
@@ -104,25 +104,6 @@ def to_representation(self, obj):
         return f"{obj.app_label}.{obj.model}"
 
 
-class ObjectTypeField(serializers.CharField):
```

Comment:
I don't know why this stopped working when we switched to `fields = "__all__"`, but changing it to a SerializerMethodField seems good enough to get it included again - unfortunately this seems to have made drf-spectacular unhappy, so I'll need to change it back and figure out what's going on instead.